importer/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@xata.io/importer",
+  "version": "0.0.0",
+  "description": "Importer for xata.io",
+  "main": "./dist/index.js",
+  "type": "module",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "tsc -p tsconfig.build.json",
+    "prepack": "npm run build",
+    "prebuild": "node scripts/add-version-file.js",
+    "postbuild": "node scripts/inject-hashbang.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/xataio/client-ts.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/xataio/client-ts/issues"
+  },
+  "homepage": "https://github.com/xataio/client-ts/blob/main/importer/README.md",
+  "dependencies": {
+    "@xata.io/client": "^0.5.0",
+    "camelcase": "^6.3.0",
+    "commander": "^9.2.0",
+    "cross-fetch": "^3.1.5",
+    "csvtojson": "^2.0.10",
```

Comment:
Not 100% sure about this dependency, seems not actively maintained

<img width="792" alt="Screenshot 2022-05-04 at 08 37 34" src="https://user-images.githubusercontent.com/2181866/166633074-e6c7fc66-a640-4f30-be45-5471d006989f.png">